cme/protocols/smb/atexec.py

```diff
@@ -203,10 +203,10 @@ def execute_handler(self, command, fileless=False):
                         break
                     except Exception as e:
                         if tries >= self.__tries:
-                            self.logger.fail(f'ATEXEC: Couldn\'t retrieve output file, maybe got detected by AV. Please increase the number of tries with the option "--get-output-tries". If it\'s still failing, try the wmi protocol or another exec method')
+                            self.logger.fail(f"ATEXEC: Couldn't retrieve output file, maybe got detected by AV. Please increase the number of tries with the option \"--get-output-tries\". If it's still failing, try the wmi protocol or another exec method")
                             break
                         if str(e).find("STATUS_BAD_NETWORK_NAME") >0 :
-                            self.logger.fail(f'ATEXEC: Get ouput failed, target has blocked ADMIN$ access (maybe command executed!)')
+                            self.logger.fail(f"ATEXEC: Get ouput failed, target has blocked ADMIN$ access (maybe command executed!)")
```

Comment:
"ouput" mispelling - should be "output"

Reply:
Also, why is "ADMIN$" hardcoded?

Reply:
Not sure, i think because it tries to explicitly read the $Admin share in line 202

Reply:
> Also, why is "ADMIN$" hardcoded?

The reason why is hardcoded here because I meet a weird situation in real-world attack, I am already hight privilege users but I can’t doing psexec/smbexec, these execute method always using “admin$”, but the weird thing is I can use atexec(pipe)

I have post some screenshot before mpgn remove his fork

Reply:
Can we reference the actual share here?

Reply:
or rather, if it's ADMIN$, make it variable and use that instead

Reply:
> or rather, if it's ADMIN$, make it variable and use that instead

Yes, you're right

Reply:
@Marshall-Hallenbeck How can I do coding in this commit XD.